### ISSUE TITLE
[android] Automaticaly open user native language entry when adding place to OSM

### ISSUE THREAD

**Issue by ioctl-user** [organicmaps/organicmaps#8435]
This request is about situation when local place language is not the same as locale language.

While adding place to OSM using Android version one should tap "Add language" button to add place name on it's native language.

It would be more handy if Organic Maps in addition to local place language will open entry for user's native language (by device locale) without additional actions .

**Comment by biodranik** [comment]
An important moment is that the user may configure several languages he/she speaks in system settings. And a proper approach may display all of them.

A disadvantage is that more vertical space will be taken by rarely used/needed translations. Maybe some tappable hints (e.g. "tap to add English translation") can be displayed, which show the cell with the language on a tap?

**Comment by ioctl-user** [comment]
Looks like single tap is a little bit more complex in terms of user friendliness, that just swiping big list down. 
At that, I could assume, most users will have no more than two languages configured.

And also ordinary user can't write local language place name if it's not one of his device language. So, default open entry in local place language looks not very useful.

As for me, the best option is showing local language and all users languages.
But adding just a single tap to open all this languages also looks as significant usability improvement.

**Comment by biodranik** [comment]
> And also ordinary user can't write local language place name if it's not one of his device language. So, default open entry in local place language looks not very useful.

That is not true. Anyone can write a name in local language if it is seen by the eyes, and if there is a necessary keyboard/alphabet to type. Modern devices allow to take a photo and then copy-paste the text from a photo.
